_includes/selector.html

```diff
@@ -657,6 +660,9 @@
                     if (this.active_additional_packages.includes("TensorFlow")) {
                         notes = [...notes, "TensorFlow requires CUDA 12."];
                     }
+                    if (this.active_additional_packages.includes("NetworkX + nx-cugraph")) {
+                        notes = [...notes, "Selecting this option will install both standard NetworkX and a GPU-accelerated backend by RAPIDS"];
```

Comment:
I don't think we need notes here, the button label "NetworkX + nx-cugraph" is clear enough to me.

Reply:
I agree with @bdice here after seeing it.  

Although, another option which is closer to a suggestion Nick made earlier and addresses a comment I made above, would be to change the button to just say "NetworkX" and then make the note say:

> This option installs NetworkX which will be GPU accelerated by the nx-cugraph backend included in RAPIDS.

Although I don't feel strongly either way, and perhaps just having the button say "NetworkX + nx-cugraph" is adequate and not confusing.